data/json/items/gunmod/magazine.json

```diff
@@ -12,13 +12,15 @@
     "symbol": ":",
     "color": "green",
     "location": "magazine",
-    "mod_targets": [ "rifle" ],
-    "acceptable_ammo": [ "223", "308" ],
+    "mod_targets": [ "rifle", "launcher" ],
+    "acceptable_ammo": [ "223", "308", "50", "40mm" ],
```

Comment:
This gunmod is installed in a "magazine" slot, and such guns shouldn't have this slot in the first place.